src/entities/mellowLpRouter.ts

```diff
@@ -115,35 +106,18 @@ class MellowLpRouter {
     const tokenAddress = await mellowRouterContract.token();
     const tokenContract = new Contract(tokenAddress, IERC20MinimalABI, this.provider);
 
-    console.log('token address:', tokenAddress);
-
     // erc20rootvault addresses
     let ERC20RootVaultAddresses: string[] = await mellowRouterContract.getVaults();
-    this.erc20RootVaultInfo = Array.from(ERC20RootVaultAddresses.keys()).map(
-      (index: number) =>
-        <ERC20RootVaultInfo>{
-          index,
-          subvaultsCount: 0, // populated below
-        },
-    );
 
     if (!isUndefined(this.pivot)) {
       ERC20RootVaultAddresses = ERC20RootVaultAddresses.slice(this.pivot, this.pivot + 1);
-      this.erc20RootVaultInfo = this.erc20RootVaultInfo.slice(this.pivot, this.pivot + 1);
     }
 
     // Map the addresses so that each of them is instantiated into a contract
     const erc20RootVaultContracts = ERC20RootVaultAddresses.map(
       (address: string) => new ethers.Contract(address, Erc20RootVaultABI, this.provider),
     );
 
-    // Populate subvaults count
```

Comment:
Let's load this data on-demand (i.e. on withdrawal/rollover). Otherwise we load it every time we open the app for example.